src/Sylius/Bundle/AdminBundle/Resources/views/Crud/create.html.twig

```diff
@@ -20,3 +20,16 @@
 
 {{ sonata_block_render_event(event_prefix ~ '.after_content', {'resource': resource}) }}
 {% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+
+    {{ sonata_block_render_event(event_prefix ~ '.javascript') }}
+{% endblock %}
+
+
+{% block stylesheets %}
+    {{ parent() }}
+
+    {{ sonata_block_render_event(event_prefix ~ '.javascript') }}
```

Comment:
Stylesheets?

1. Let's use plural names - they match the block convention.
2. First stylesheets, then javascripts.